frontend/src/__mocks__/mockNimResource.ts

```diff
@@ -45,11 +45,20 @@ export const mockNimImages = (): ConfigMapKind =>
     },
   });
 
-export const mockNimInferenceService = (): InferenceServiceKind => {
+type NimInferenceService = {
+  namespace?: string;
+  displayName?: string;
+};
+
+export const mockNimInferenceService = ({
+  displayName = 'Test Name',
+  namespace = 'test-project',
+}: NimInferenceService): InferenceServiceKind => {
```

Comment:
```suggestion
}: NimInferenceService = {}): InferenceServiceKind => {
```

Seems if you did this you'd avoid a bunch of unnecessary changes